lib/s3_helper.rb

```diff
@@ -131,7 +131,7 @@ def update_lifecycle(id, days, prefix: nil, tag: nil)
   end
 
   def update_tombstone_lifecycle(grace_period)
-    return if @tombstone_prefix.blank?
+    return if not SiteSetting.s3_configure_tombstone_policy or @tombstone_prefix.blank?
```

Comment:
`if not` is and `or` are not something we usually use

I would probably do

return if !SiteSetting.s3_configure_tombstone_policy
return if @tombstone_prefix.blank? 

or 


return if !SiteSetting.s3_configure_tombstone_policy || @tombstone_prefix.blank?

Reply:
my bad, that is much better, will be more careful after this :+1: